finality-aleph/src/environment.rs

```diff
@@ -247,55 +226,67 @@ where
     // Pulls requests from the priority queue (sorted by scheduled time) and sends them to random peers
     // as long as they are scheduled at time <= curr_time
     pub(crate) fn trigger_tasks(&mut self) {
-        loop {
+        while let Some(request) = self.requests.peek() {
```

Comment:
This equivalent approach to this loop seems more readable to me

Reply:
Right.